doc/adr/0000-use-markdown-architectural-decision-records.md

```diff
@@ -0,0 +1,26 @@
+# Use Markdown Architectural Decision Records
```

Comment:
Do we need to have an ADR for this PR?

Reply:
I think it makes sense to start off this way. One thing we should think about still is wether to add [ADR-xxxx] into the title or let sphinx do the numbering based on the filenames. I almost then to sphinx numbering because it avoids typos.

Reply:
Yes I agree with the ADR-xxxx. Downside: it has to be done manually each time. Upside: the number is hardcoded in the document, so hard to temper with.

Reply:
In that case we should indeed remove the sphinx numbering in the toc table

Reply:
Personally, I use the [Rewrap extension](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) in VScode. Note that we already have a setting specified for that extension here:
https://github.com/ComPWA/expertsystem/blob/cf1728eedf466f2d590a6eee77ddfb2b94359b76/.vscode/settings.json#L29
The question is how to enforce this in a more general way, through a pre-commit hook for instance.

Reply:
Okay I think we should forget about line wrapping.
1. Too much effort to enforce this, even if there are tools available
2. GitHub offers several [diff view options](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-comparing-branches-in-pull-requests#diff-view-options), so there are plenty of ways to conveniently review an ADR that does not have wrapped lines.
3. PR review in VScode does not have to be done through a diff editor, so you have soft wrapping at your disposal there.